clickhouse-admin/api/src/lib.rs

```diff
@@ -33,9 +33,9 @@ pub trait ClickhouseAdminKeeperApi {
     /// configuration set via this endpoint.
     #[endpoint {
         method = PUT,
-        path = "/config",
+        path = "/gen-config-and-enable",
```

Comment:
I don't mind changing the name of the method, but the endpoint name should not change. Endpoint names should always be nouns as they define resources in REST. We are putting a `config` which is idempotent. What internal side-effects it has are irrelevant to the API.

It should also be noted that only the first time this endpoint is run on a svc does it transition from disabled to enabled. Every other time we are just updating the config.

Reply:
hahahahhahahah I feel like you left me this exact same feedback somewhere else 😄